internal/tiltfile/docker.go

```diff
@@ -25,7 +25,7 @@ type dockerImage struct {
 	baseDockerfile     dockerfile.Dockerfile
 	configurationRef   container.RefSelector
 	deploymentRef      reference.Named
-	syncs              []sync
+	syncs              []pathSync
```

Comment:
Are these changes supposed to be here?

Reply:
yes. you can't have an `import` and a `type` with the same name. the go compiler yells at you

Reply:
Ah, gotcha makes sense